news/_posts/core-weekly/2021-05-03-coreweekly-17-2021.md

```diff
@@ -0,0 +1,184 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 17 of 2021"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2021-05-03
+authors: [ PrestaShop ]
+image: /assets/images/2017/04/core_weekly_banner.jpg
+twitter_image: /assets/images/theme/banner-core-weekly.jpg
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 19th to Sunday 25th of April 2021.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+## General messages
+
+[write a nice message here, or remove the "General messages" section]
```

Comment:
Dear developers,

[PrestaShop 1.7.7.4](https://build.prestashop.com/news/prestashop-1-7-7-4-maintenance-release/) was released on the 3rd of May, 2021. This maintenance release fixes 13 bugs reported on previous 1.7.7 versions.

## Releases

* [PrestaShop](https://github.com/PrestaShop/PrestaShop) [1.7.7.4](https://github.com/PrestaShop/PrestaShop/releases/tag/1.7.7.4)